spec/oauth/oidc_spec.lua

```diff
@@ -227,6 +228,58 @@ describe('OIDC', function()
       local credentials, _, _, err = oidc:transform_credentials({ access_token = access_token })
       assert(credentials, err)
     end)
+
+    describe('with app_id modification', function()
```

Comment:
I think this could be more descriptive. The requirement is not to modify the app_id is to get it from a custom JWT claim.